### DISCUSSION TITLE
Important Update: Discontinuation of Free TURN Services

### DISCUSSION THREAD

**Discussion by jonasgloning** [peers/peerjs#1172] [Announcements]
Dear PeerJS Community,

We regret to inform you that _we can no longer offer free TURN services_ going forward.

Recent weeks have seen a surge in attacks and misuse, leading to a situation where we lack the necessary funding and personnel to maintain this service. We understand that this change may impact small projects that previously relied on these services, and they may now face the same challenges we have encountered. Unfortunately, this decision is final, and we are very sorry that we could not provide advance notice.

TURN servers are necessary for only a subset of users, and it's worth noting that for many smaller hobby projects, a TURN server may not be required at all.

If you are aware of alternative TURN service providers, whether free or paid, please share them in the comments. As a starting point, you might consider [Open Relay](https://www.metered.ca/tools/openrelay/), although we have not personally vetted their service. Alternatively, you can look into [Coturn](https://github.com/coturn/coturn) for self-hosting options.

Thank you for your understanding.

Jonas

---

Please note that this does not affect the PeerServer public instance ([0.peerjs.com](https://0.peerjs.com)), which will remain operational.

**Comment by therealPaulPlay** [comment]
Thank you for the announcement! Maybe consider updating the API Docs, as they right now claim that "The PeerServer cloud service provides a free TURN server", just to avoid confusion. PeerJS is really lovely by the way! Amazing in terms of usability and integration and the docs are super straightforward :)

**Comment by SeanBannister** [comment]
The docs https://peerjs.com/docs/ still mention "The PeerServer cloud service provides a free TURN server"
